pkg/common/common.go

```diff
@@ -216,3 +217,14 @@ func StripDefaultPorts(fromUrl string) (string, error) {
 	}
 	return r.URL.String(), nil
 }
+
+// GetImagePullPolicy get imagePullPolicy for a container, based on its image.
+// If image contains a digest, use IfNotPresent; otherwise, Always.
+// Reference: https://github.com/distribution/distribution/blob/v2.7.1/reference/reference.go
+func GetImagePullPolicy(image string) corev1.PullPolicy {
+	if strings.Contains(image, "@") {
```

Comment:
To reinforce this logic lets check something like `@sha256` or a regex expression like `@sha256:[a-f0-9]{64}`

Reply:
my fear of being more focused in sha256 here is ignoring another algorithms and digests, or that's ok?

Reply:
@rayfordj would be the authority here, but afaik our images are only shipped w/ @sha256

Reply:
@weshayutin , I believe at this time, it should be relatively safe to be more explicit in the match, as @shubham-pampattiwar suggests, to reinforce the logic.  We -- all of Red Hat using digests -- are currently embedded with sha256 digests for pinned images; in the event image digest does eventually move away from sha256 to something else, we should have very sufficient runway to accommodate.  

@mateusoliveira43 , if it eases your concern, it might be that a regex offers more flexibility for adjustment in the future to match other such algorithms & digests should it be required versus "is `@sha256` in string".  I defer to you.

Reply:
It's times like these I'm reminded of [this](https://xkcd.com/1171/):
![](https://imgs.xkcd.com/comics/perl_problems.png)

Reply:
alright, going to update!

Reply:
done